src/algorithms/rsa.rs

```diff
@@ -245,6 +245,58 @@ pub fn recover_primes(n: &BigUint, e: &BigUint, d: &BigUint) -> Result<(BigUint,
     Ok((p, q))
 }
 
+/// Compute the modulus of a key from its primes.
+pub(crate) fn compute_modulus(primes: &[BigUint]) -> BigUint {
+    let mut n = BigUint::one();
+
+    for prime in primes {
+        n *= prime;
+    }
+
+    n
```

Comment:
I would think this would work:

```suggestion
    primes.iter().product()
```